src/images/garden-icon.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="46px" height="46px" viewBox="0 0 46 46" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 62 (91390) - https://sketch.com -->
+    <title>Path</title>
+    <desc>Created with Sketch.</desc>
+    <g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="garden-horizontal" transform="translate(-1.000000, -2.000000)" fill="#5DAE90">
+            <path d="M1.73004027,24.6616797 C1.73004027,37.096832 12.0988594,47.2164732 24.8402049,47.2164732 C37.5815504,47.2164732 47.8624982,37.096832 47.8624982,24.6616797 C47.8624982,24.6616797 47.8624982,2.10688624 47.8624982,2.10688624 C35.1211527,2.10688624 24.7523335,12.2265274 24.7523335,24.6616797 C24.7523335,12.2265274 14.3835144,2.10688624 1.64216892,2.10688624 C1.73004027,2.10688624 1.73004027,24.6616797 1.73004027,24.6616797 Z" id="Path"></path>
+        </g>
+    </g>
+</svg>
```

Comment:
This should be getting sourced from `svg-icons`. (Maybe that is planned as a follow-up)

Reply:
Didn't realize we had a 26px variant of this! I've moved to importing it from there.